staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1794,11 +1794,6 @@ type EphemeralVolumeSource struct {
 	//
 	// Required, must not be nil.
 	VolumeClaimTemplate *PersistentVolumeClaimTemplate `json:"volumeClaimTemplate,omitempty" protobuf:"bytes,1,opt,name=volumeClaimTemplate"`
-
-	// Specifies a read-only configuration for the volume.
-	// Defaults to false (read/write).
-	// +optional
-	ReadOnly bool `json:"readOnly,omitempty" protobuf:"varint,2,opt,name=readOnly"`
```

Comment:
comment out rather than delete, and add a tombstone comment so we never reuse this json field name or proto tag. see other "tombstoned" comments in types.go files for examples.

Reply:
Done.